### ISSUE TITLE
App crash on markText using iOS

### ISSUE THREAD

**Issue by radkra** [JimmyDaddy/react-native-image-marker#105]
**Describe the bug**
IOS application crashes while I'm trying to use markText functionality (markImages works like a charm tho). I'm pasting below the code which makes the crash. I was checking similar threads in issues, but cannot see what I'm doing wrong.

```
Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
Exception Subtype: UNKNOWN_0xD at 0x0000000000000000
Exception Codes: 0x000000000000000d, 0x0000000000000000
```

**To Reproduce**
```
 ImageMarker.markText({
      src: storyBase,
      text: 'text',
      saveFormat: ImageFormat.png,
      color: '#FFFFFF',
      fontName: 'Arial-Regular',
      fontSize: 12,
      X: 0,
      Y: 0,
      shadowStyle: null,
      textBackgroundStyle: null,
      scale: 1, 
      quality: 100, 
    })
      .then(path => {
        console.log('path' + path);
      })
      .catch(error => {
        console.log('error', error);
      });

```

**Devlopment environment(please complete the following information):**
 - OS: MacOS
 - nodejs:  18.16.1
 - react-native: 0.71.3
 - react-native-image-marker :  ^0.9.1

**Comment by LaGregance** [comment]
Same problem for me, in the xcode console I get:

```
2023-07-11 14:20:38.166826+0200 MyApp[420:14060] [native] Exception '*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[2]' was thrown while invoking addTextByPosition on target ImageMarker with params (
        {
        color = red;
        fontName = Poppins;
        fontSize = 18;
        maxSize = 2048;
        position = bottomRight;
        quality = 100;
        saveFormat = jpg;
        scale = 1;
        src =         {
            "__packager_asset" = 0;
            uri = "/var/mobile/Containers/Data/Application/BDD3F61B-1169-4468-90BD-F65D1527359E/tmp/DB60F0B8-E835-4456-B760-D2E6BF198D8E.jpg";
        };
        text = "11/07/2023 14:20 iPad Mini 5";
    },
    18032,
    18033
)
```

Using the code:
```
await ImageMarker.markText({
    src,
    text: '11/07/2023 14:20 iPad Mini 5',
    position: Position.bottomRight,
    color: '#FF0000',
    fontSize: 18,
    fontName: 'Poppins',
    scale: 1,
    quality: 100,
    saveFormat: ImageFormat.jpg,
  });
```

**Comment by JimmyDaddy** [comment]
fixed it on latest version: 0.9.2, pls update it @LaGregance @radkra

**Comment by LaGregance** [comment]
Thanks, you are the best !
